src/WebJobs.Extensions.DurableTask/Listener/OutOfProcOrchestrationShim.cs

```diff
@@ -104,6 +116,149 @@ internal async Task<bool> ScheduleDurableTaskEvents(OrchestrationInvocationResul
+                        }
+                    }
+
+                    break;
+                case AsyncActionType.CallActivityWithRetry:
+                    task = this.context.CallActivityWithRetryAsync(action.FunctionName, action.RetryOptions, action.Input);
+                    break;
+                case AsyncActionType.CallSubOrchestrator:
+                    task = this.context.CallSubOrchestratorAsync(action.FunctionName, action.InstanceId, action.Input);
+                    break;
+                case AsyncActionType.CallSubOrchestratorWithRetry:
+                    task = this.context.CallSubOrchestratorWithRetryAsync(action.FunctionName, action.RetryOptions, action.InstanceId, action.Input);
+                    break;
+                case AsyncActionType.CallEntity:
+                    {
+                        var entityId = EntityId.GetEntityIdFromSchedulerId(action.InstanceId);
+                        task = this.context.CallEntityAsync(entityId, action.EntityOperation, action.Input);
+                        break;
+                    }
+
+                case AsyncActionType.SignalEntity:
+                    {
+                        // We do not add a task because this is 'fire and forget'
+                        var entityId = EntityId.GetEntityIdFromSchedulerId(action.InstanceId);
+                        this.context.SignalEntity(entityId, action.EntityOperation, action.Input);
+                        task = fireAndForgetTask;
+                        break;
+                    }
+
+                case AsyncActionType.ScheduledSignalEntity:
+                    {
+                        // We do not add a task because this is 'fire and forget'
+                        var entityId = EntityId.GetEntityIdFromSchedulerId(action.InstanceId);
+                        this.context.SignalEntity(entityId, action.FireAt, action.EntityOperation, action.Input);
+                        task = fireAndForgetTask;
+                        break;
+                    }
+
+                case AsyncActionType.ContinueAsNew:
+                    this.context.ContinueAsNew(action.Input);
+                    task = fireAndForgetTask;
+                    break;
+                case AsyncActionType.WaitForExternalEvent:
+                    task = this.context.WaitForExternalEvent<object>(action.ExternalEventName);
+                    break;
+                case AsyncActionType.CallHttp:
+                    task = this.context.CallHttpAsync(action.HttpRequest);
+                    break;
+                case AsyncActionType.WhenAll:
+                    task = Task.WhenAll(mapInvokeAPIFromAction.Invoke(action.CompoundActions));
+                    break;
+                case AsyncActionType.WhenAny:
+                    task = Task.WhenAny(mapInvokeAPIFromAction.Invoke(action.CompoundActions));
+                    break;
+                default:
+                    break;
```

Comment:
I know that this is existing from before, but at some point we may want to reconsider our silent handling of the default case.

Reply:
Good call, I actually would like to fix it _now_, I want to avoid having silent errors moving forward, as part of improving confidence in our ability to reason about OOProc behavior. I'll push something.